tool/jt.rb

```diff
@@ -131,8 +131,13 @@ def mvn(*args)
     sh 'mvn', *args
   end
 
-  def mspec(command, *args)
-    sh 'ruby', 'spec/mspec/bin/mspec', command, '--config', 'spec/truffle/truffle.mspec', *args
+  def mspec(env, command, *args)
+    env_vars = {}
+    mspec_env env_vars, command, *args
```

Comment:
`env` is not used in the definition here and this will ignore the first argument. I think you meant to leave the arguments unchanged.